src/tests/DoctorRegistration.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
I would recommend you remove the test file for now and make it when you finish the whole steppers.

Reply:
removed